components/error_code/src/backup_stream.rs

```diff
@@ -11,6 +11,10 @@ define_error_codes! {
         "A task not found.",
         "Please check the spell of your task name."
     ),
+    OUT_OF_QUOTA => ("OutOfQuota",
+        "Some of quota has been exceed, hence the task cannot continue.",
+        "For memory quotas, please check whether there are huge transactions. You may also increase the quota by modifying config."
```

Comment:
Could you add the config name?

Reply:
For now there isn't an universal memory quota config flag for log backup and I'm going to unify the current two memory quota configs. I think we can add that config name then or once the config term changed I'm afraid this may confuse the user.